hoot-core/src/main/cpp/hoot/core/algorithms/alpha-shape/AlphaShape.cpp

```diff
@@ -235,6 +236,10 @@ void AlphaShape::insert(const vector<pair<double, double>>& points)
     _pDelauneyTriangles->insert(randomized[i].first, randomized[i].second);
   }
   LOG_VARD(_pDelauneyTriangles->getFaces().size());
+  //  Report the final progress
+  PROGRESS_INFO(
+    "Added " << StringUtils::formatLargeNumber(randomized.size() - 1) << " / " <<
+    StringUtils::formatLargeNumber(randomized.size() - 1) << " points.");
```

Comment:
In the logs I like to see the final `Added 12101 / 12101 points.` instead of the last thing being `Added 12000 / 12101 points.`